src/core/pyproxy.ts

```diff
@@ -938,6 +1026,43 @@ export class PyProxyIteratorMethods {
   }
 }
 
+export class PyProxyAsyncIteratorMethods {
+  async next(arg: any = undefined): Promise<IteratorResult<any, any>> {
```

Comment:
It seems like `PyProxy.next` is shown twice in docs (async and sync version), but I'm not sure what we should do. So I'm okay to merge.

Reply:
I think we can add `/** @private */` to one.

Reply:
In the long run it'd probably be good to break up the `PyProxy` docs like the way I did to the `JsProxy` docs in #3277.

Reply:
> I think we can add `/** @private */` to one.

That makes sense. Thanks!

Reply:
Of course the docs aren't strictly accurate since `next` might instead translate to Python `anext`. But it is okay.